src/shared/components/palette/address/write-address-field.component.ts

```diff
@@ -72,6 +76,13 @@ export class WriteAddressFieldComponent extends AbstractFieldWriteComponent impl
           );
         });
       this.addressList.setValue(undefined);
+      this.focusAddressList();
+    }
+  }
+
+  focusAddressList(): void {
```

Comment:
Giri can you explain what has been done here please

Reply:
After each search the address list needs to be focused. The directive FocusElementDirective, focuses the element only the first time when the views content is initialised. For any subsequent searches, it wasn't focusing. There may be better alternatives to this..